src/Components/Pages/Playlist/PlaylistPage.tsx

```diff
@@ -2,29 +2,32 @@ import React from "react"
 import { Container } from "react-bootstrap"
 import { match } from "react-router-dom"
 import PlaylistService from "../../../Services/PlaylistService"
-import { PlaylistObject } from "../../../Models/SpotifyObjects/PlaylistObjects"
+import { PlaylistObject, SimplifiedPlaylistObject } from "../../../Models/SpotifyObjects/PlaylistObjects"
 import Navbar from "../../Shared/Navbar"
 import TrackTable from "../../Shared/TrackList/TrackTable"
 import PlaylistHeader from "./PlaylistHeader"
 import PlaylistZeroState from "./PlaylistZeroState"
+import SearchBar from "../../Shared/SearchBar"
+import { getRequest } from "../../../Endpoints/AxiosConfig"
 
 interface PlaylistPageProps {
-    match: match<{id: string}>
+    match?: match<{ id: string }>
```

Comment:
Why is this optional? From what I understand, the match should always be defined even if there aren't any params